### PR TITLE
Initial codes

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Initial codes about implementing `array_key_first` and `array_key_last` function polyfills.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,5 +9,11 @@
         "phpunit/phpunit": "^4.8"
     },
     "license": "MIT",
-    "minimum-stability": "stable"
+    "minimum-stability": "stable",
+    "autoload": {
```
Comment (Ayesh) [inline composer.json:13]: Can we please make both functions reside in the same file?

**Review by Ayesh — COMMENTED** [review]
Thank you. They all look good. I would like to suggest that we keep both functions in the same file to keep things simple and have one fewer file to autoload.

File: composer.json (hunk 1 above)
Comment (peter279k) [reply composer.json:13]: @Ayesh, sorry for my slow response.

I think it will be the `array_key_first_last.php` file?

Comment (Ayesh) [reply composer.json:13]: I'm sorry for the late review :)
`array_key_first_last.php` sounds good to me too. Thank you!

Comment (peter279k) [reply composer.json:13]: I will do that on next PR :).

**Pull request merged**